app/views/labware/_asset_summary.html.erb

```diff
@@ -100,5 +100,17 @@
       <td><%= asset.storage_location %></td>
     </tr>
   <% end %>
+  <tr>
+    <th>Retention Instruction</th>
+    <!-- First, check if retention instructions are there. -->
+    <% if asset.retention_instruction.blank? %>
+        <td>Not specified</td>
+    <!-- If retention instructions are present, display them. -->
+    <%  else  %>
+      <td><%= find_retention_instruction_from_key(asset.retention_instruction.to_sym) %></td>
+    <%  end %>
+    <!-- There shouldn't be a situation where retention instructions are there in both custom_metadata -->
+    <!-- and in labware table's retention instruction attribute -->
```

Comment:
Is this comment still meant to be here, after removing the backwards compatible code around custom metadata?